diagnostic.md

```diff
@@ -48,11 +49,36 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+
+    ```js
+    export default Ember.Route.extend({
+    model: function(params){
+    return [
+      {
+        id: 123,
+        name: 'movie one',
+      },
+
+      {
+        id: 124,
+        name: 'movie one',
+      },
+      ][params.product_id - 1];
+      }
+    });
+
     ```
```

Comment:
Close! The idea here is that 123 is stored in the route params when someone navigates to /movies/123. Because the route is /movies/:movie_id, movie_id becomes a route param.

We can access route params via the `params` parameter. We'd access the value 123 with `params.movie_id`